app/views/course/statistics/staff.html.slim

```diff
@@ -0,0 +1,12 @@
+= page_header
+table.table.table-middle-align.table-hover
+  thead
+    tr
+      th = t('.name')
+      th = t('.marked')
+      th = t('.students')
+      th = t('.average_time')
+      th = t('.stddev')
+  tbody
+    - @staffs.each do |staff|
```

Comment:
I didn't do that because the variable name will become `staff_performance` instead.